### PR TITLE
Release 1.0.0-rc.2

### PR DESCRIPTION
**Pull request by Kielek**
## Why &  What

Release 1.0.0-rc.2
Drop support for Azure SDK on .NET Framework (see failing test details below in discussion).

## Tests

* [x] CI
* [x] MacOS with Linux Containers
* [x] Windows with Linux Containers (after excluding Azure SDK on .NET Framework)

## Checklist

- [x] `CHANGELOG.md` is updated.
- [x] Documentation is updated.
- ~~[ ] New features are covered by tests.~~

### CODE REVIEW

**Review by pellared — APPROVED** [review]

**Comment by pellared** [comment]
`nuke Workflow` passed on macOS

**Review by pjanotti — APPROVED** [review]

**Review by rajkumar-rangaraj — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,7 +5,21 @@ All notable changes to this component are documented in this file.
...
+### Added
+
+### Changed
+
+### Deprecated
+
+### Removed
+
+### Fixed
+
```
Comment (pjanotti) [inline CHANGELOG.md:19]: Please, include a blurb for #2780

**Review by nrcventura — APPROVED** [review]

**Comment by pellared** [comment]
Adding more output for the test failure from my machine

```
19:55:22 [ERR] [xUnit.net 00:00:59.25]     IntegrationTests.AzureTests.SubmitsTraces(packageVersion: "12.17.0") [FAIL]
19:55:22 [ERR] [xUnit.net 00:00:59.26]     IntegrationTests.AzureTests.SubmitsTraces(packageVersion: "12.13.0") [FAIL]
19:55:23 [DBG]   Failed IntegrationTests.AzureTests.SubmitsTraces(packageVersion: "12.17.0") [1 ms]
19:55:23 [DBG]   Error Message:
19:55:23 [DBG]    Docker.DotNet.DockerApiException : Docker API responded with status code=InternalServerError, response={"message":"driver failed programming external connectivity on endpoint azure-storage-54324 (befba1179e06ddf0249edff67a2abab3ae2ed3b7d259379b8c56fde65d983f60): listen tcp4 0.0.0.0:54324: bind: address already in use"}
19:55:23 [DBG]
19:55:23 [DBG]   Stack Trace:
19:55:23 [DBG]      at Docker.DotNet.DockerClient.<HandleIfErrorResponseAsync>d__67.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at Docker.DotNet.DockerClient.<MakeRequestAsync>d__55.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at Docker.DotNet.ContainerOperations.<StartContainerAsync>d__16.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Clients.TestcontainersClient.<StartAsync>d__28.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Containers.DockerContainer.<UnsafeStartAsync>d__61.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Containers.DockerContainer.<StartAsync>d__49.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at IntegrationTests.AzureFixture.<LaunchAzureContainerAsync>d__9.MoveNext() in C:\Users\rpajak\repos\opentelemetry-dotnet-instrumentation\test\IntegrationTests\AzureCollection.cs:line 69
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at IntegrationTests.AzureFixture.<InitializeAsync>d__7.MoveNext() in C:\Users\rpajak\repos\opentelemetry-dotnet-instrumentation\test\IntegrationTests\AzureCollection.cs:line 47
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]   Failed IntegrationTests.AzureTests.SubmitsTraces(packageVersion: "12.13.0") [1 ms]
19:55:23 [DBG]   Error Message:
19:55:23 [DBG]    Docker.DotNet.DockerApiException : Docker API responded with status code=InternalServerError, response={"message":"driver failed programming external connectivity on endpoint azure-storage-54324 (befba1179e06ddf0249edff67a2abab3ae2ed3b7d259379b8c56fde65d983f60): listen tcp4 0.0.0.0:54324: bind: address already in use"}
19:55:23 [DBG]
19:55:23 [DBG]   Stack Trace:
19:55:23 [DBG]      at Docker.DotNet.DockerClient.<HandleIfErrorResponseAsync>d__67.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at Docker.DotNet.DockerClient.<MakeRequestAsync>d__55.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at Docker.DotNet.ContainerOperations.<StartContainerAsync>d__16.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Clients.TestcontainersClient.<StartAsync>d__28.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Containers.DockerContainer.<UnsafeStartAsync>d__61.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at DotNet.Testcontainers.Containers.DockerContainer.<StartAsync>d__49.MoveNext()
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at IntegrationTests.AzureFixture.<LaunchAzureContainerAsync>d__9.MoveNext() in C:\Users\rpajak\repos\opentelemetry-dotnet-instrumentation\test\IntegrationTests\AzureCollection.cs:line 69
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
19:55:23 [DBG]    at IntegrationTests.AzureFixture.<InitializeAsync>d__7.MoveNext() in C:\Users\rpajak\repos\opentelemetry-dotnet-instrumentation\test\IntegrationTests\AzureCollection.cs:line 47
19:55:23 [DBG] --- End of stack trace from previous location where exception was thrown ---
19:55:23 [DBG]    at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
19:55:23 [DBG]    at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
```

**Comment by pjanotti** [comment]
🤔 
> `[ERR] RunManagedIntegratio: [xUnit.net 00:08:27.11]     IntegrationTests.DomainNeutralTests.SubmitsTraces [FAIL]` - passed after rerun.

Locally or in CI? This test is not flaky IIRC, let me know if you have detailed error messages.

**Comment by Kielek** [comment]
> 🤔
> 
> > `[ERR] RunManagedIntegratio: [xUnit.net 00:08:27.11]     IntegrationTests.DomainNeutralTests.SubmitsTraces [FAIL]` - passed after rerun.
> 
> Locally or in CI? This test is not flaky IIRC, let me know if you have detailed error messages.

Locally.

**Comment by Kielek** [comment]
As agreed yesterday with @rajkumar-rangaraj , I am dropping support to Azure SDK on .NET Framework. We can bring it back in next release.

File: CHANGELOG.md (hunk 1 above)
Comment (Kielek) [reply CHANGELOG.md:19]: Done.

**Review by lachmatt — APPROVED** [review]

**Pull request merged**